components/dashboard/component.go

```diff
@@ -0,0 +1,213 @@
+package dashboard
+
+import (
+	"context"
+	"errors"
+	"net"
+	"net/http"
+	"runtime"
+	"strconv"
+	"time"
+
+	"github.com/labstack/echo/v4"
+	"github.com/labstack/echo/v4/middleware"
+	"github.com/labstack/gommon/log"
+	"go.uber.org/dig"
+
+	"github.com/iotaledger/hive.go/app"
+	"github.com/iotaledger/hive.go/autopeering/peer"
+	"github.com/iotaledger/hive.go/autopeering/peer/service"
+	"github.com/iotaledger/iota-core/pkg/daemon"
+	"github.com/iotaledger/iota-core/pkg/network/p2p"
+	"github.com/iotaledger/iota-core/pkg/protocol"
+)
+
+func init() {
+	Component = &app.Component{
+		Name:      "Dashboard",
+		DepsFunc:  func(cDeps dependencies) { deps = cDeps },
+		Params:    params,
+		Configure: configure,
+		Run:       run,
+		IsEnabled: func(c *dig.Container) bool {
+			return true
```

Comment:
I think we should have a parameter to disable it, just like in the coreapi plugin